tests/openedx_tagging/core/tagging/test_views.py

```diff
@@ -2399,6 +2400,39 @@ def test_import(self, file_format: str) -> None:
         for i, tag in enumerate(tags):
             assert tag["value"] == new_tags[i]["value"]
 
+    @ddt.data(
+        "csv",
+        "json",
+    )
+    def test_import_plan(self, file_format: str) -> None:
+        """
+        Tests planning import a valid taxonomy file.
+        """
+        url = TAXONOMY_TAGS_IMPORT_PLAN_URL.format(pk=self.taxonomy.id)
+        new_tags = [
+            {"id": "tag_1", "value": "Tag 1"},
+            {"id": "tag_2", "value": "Tag 2"},
+            {"id": "tag_3", "value": "Tag 3"},
+            {"id": "tag_4", "value": "Tag 4"},
+        ]
+        file = self._get_file(new_tags, file_format)
+
+        self.client.force_authenticate(user=self.staff)
+        response = self.client.put(
+            url,
+            {"file": file},
+            format="multipart"
+        )
+        assert response.status_code == status.HTTP_200_OK
```

Comment:
@rpenido Is it possible to do more verifications here? Maybe verify that it does not return error messages and that it does return data

Reply:
Done e0fe6e8eb4cd6a03042707027d225593a1875fd5!